e2e/specs/st_metric.spec.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @license
+ * Copyright 2018-2021 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("st.metric", () => {
+  before(() => {
+    cy.visit("http://localhost:3000/");
+  });
+
+  describe("Test first metric", () => {
+    it("displays the correct first label text", () => {
```

Comment:
nit: not a big deal at all, but you can probably get rid of the first/second/third within the `it` descriptions since that info is present in the `describe` already